portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +26,30 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  ArrayList<String> comments = new ArrayList<String>();
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Anthony!</h1>");
+
+	//add some hard coded values to comments list
+    comments.add("hello");
+    comments.add("world");
+    comments.add("so cool!");
+
+	//convert comments to json
+	String json = convertToJsonUsingGson(comments);
+
+	//set response to comments in json form
```

Comment:
nit: Could you fix the indentations and make it consistent with the other lines in this method?